src/main/java/io/hops/hopsworks/expat/migrations/featurestore/featuregroup/UpdateFeatureGroupFeatureDescription.java

```diff
@@ -46,7 +46,7 @@
 import org.eclipse.persistence.jaxb.JAXBContextFactory;
 import org.eclipse.persistence.jaxb.MarshallerProperties;
 import org.eclipse.persistence.oxm.MediaType;
-import org.elasticsearch.common.CheckedBiConsumer;
+import org.opensearch.common.CheckedBiConsumer;
```

Comment:
In general yes, that seems right, but I am not sure if this should be changed for old classes, cause back then we weren't using open search yet. Did you double check this with @o-alex  or @maismail ?

Reply:
talked to @maismail, should be ok.